spago.dhall

```diff
@@ -0,0 +1,15 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
```

Comment:
Shall we remove this comment?

Reply:
Sure. I'm actually wondering why that comment still exists. I think I'll open an issue.